
{'date': 'Sun Jul 12 2020 15:07:50.691 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 17
  TOPO_ESQUERDA = "AN"
  ^
IndentationError: expected an indented block
'''},
{'date': 'Sun Jul 12 2020 15:08:05.955 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 17
  TOPO_ESQUERDA = "AN"
  ^
IndentationError: expected an indented block
'''},
{'date': 'Sun Jul 12 2020 15:08:46.175 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 177
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 24
    circus(1, [[TOPO_ESQUERDA, TOPO_CENTRO, TOPO_DIREITA], [MEIO_ESQUERDA, CENTRO,
TypeError: 'module' object is not callable
'''},
{'date': 'Sun Jul 12 2020 15:10:56.694 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 177
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 24
    circus(1, [[TOPO_ESQUERDA, TOPO_CENTRO, TOPO_DIREITA], [MEIO_ESQUERDA, CENTRO,MEIO_DIREITA], [FUNDO_ESQUERDA, FUNDO_CENTRO, FUNDO_DIREITA]])
TypeError: 'module' object is not callable
'''},
{'date': 'Sun Jul 12 2020 15:11:44.673 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 177
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 17
    f = open("names.txt", "w")
IOError: File names.txt not found
'''},
{'date': 'Sun Jul 12 2020 15:15:58.76 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 177
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 37
    	Calcada().vai()
  module <module> line 28
    		self.cena = Cena(cena_praia, direita=Praia())
TypeError: 'module' object is not callable
'''},
{'date': 'Sun Jul 12 2020 15:23:17.885 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 177
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 35
    	Calcada().vai()
  module <module> line 26
    		self.cena = Cena(cena_praia, direita=Praia())
TypeError: 'module' object is not callable
'''},
{'date': 'Sun Jul 12 2020 15:25:30.91 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 177
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 36
    	f = open("names.txt", "w")
IOError: File names.txt not found
'''},
{'date': 'Sun Jul 12 2020 15:33:56.299 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 177
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 36
    	f = open("names.txt", "w")
IOError: File names.txt not found
'''},
{'date': 'Wed Jul 15 2020 00:47:04.124 GMt-0300 (Hora padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 177
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 28
    class Eventos:
  module <module> line 38
    MA
NameError: name 'MA' is not defined
'''},
{'date': 'Wed Jul 15 2020 02:02:02.310 GMt-0300 (Hora padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 177
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 28
    class Eventos:
  module <module> line 38
    matrizMapaFase = [[self.link2],
AttributeError: 'GUI' object has no attribute 'link2'
'''},
{'date': 'Wed Jul 15 2020 02:08:14.526 GMt-0300 (Hora padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 88
  
  ^
IndentationError: expected an indented block
'''},
{'date': 'Wed Jul 15 2020 02:09:49.199 GMt-0300 (Hora padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 55
  contador = 0
  ^
IndentationError: expected an indented block
'''},
{'date': 'Wed Jul 15 2020 02:38:46.270 GMt-0300 (Hora padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 76
  for i in range (3):
  ^
IndentationError: unexpected indent
'''},
{'date': 'Wed Jul 15 2020 02:39:07.516 GMt-0300 (Hora padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 83
  contador = contador + 
                         ^
SyntaxError: invalid syntax
'''},
{'date': 'Wed Jul 15 2020 02:54:57.261 GMt-0300 (Hora padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 177
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 28
    class Eventos:
  module <module> line 73
    if self.boneco.x > 100 and self.boneco.y > 100:
AttributeError: 'GUI' object has no attribute 'boneco'
'''},
{'date': 'Wed Jul 15 2020 02:55:21.796 GMt-0300 (Hora padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 177
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 28
    class Eventos:
  module <module> line 73
    if andaboneco.self.boneco.x > 100 and self.boneco.y > 100:
NameError: name 'andaboneco' is not defined
'''},
{'date': 'Wed Jul 15 2020 02:59:57.145 GMt-0300 (Hora padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 177
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 28
    class Eventos:
  module <module> line 45
    document.bind("keydown", self.anda_boneco)  # captura o evento de teclado
AttributeError: 'GUI' object has no attribute 'anda_boneco'
'''},
{'date': 'Wed Jul 15 2020 15:56:16.394 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 73
  if .self.boneco.x > 200 and self.boneco.y > 200:
      ^
SyntaxError: invalid syntax
'''},
{'date': 'Wed Jul 15 2020 16:59:12.287 GMt-0300 (Hora padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 54
  def portal (self, x, y)
                          ^
SyntaxError: invalid syntax
'''},
{'date': 'Wed Jul 15 2020 17:02:16.360 GMt-0300 (Hora padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 58
  if x > 400 and y > 200
                         ^
SyntaxError: invalid syntax
'''},
{'date': 'Wed Jul 15 2020 17:06:44.308 GMt-0300 (Hora padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 54
  ''''  def portal (self, x, y):
  ^
IndentationError: unexpected indent
'''},
{'date': 'Wed Jul 15 2020 17:24:46.439 GMt-0300 (Hora padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 90
  if self.boneco.x > 400 and self.boneco.y > 200:
  ^
IndentationError: expected an indented block
'''},
{'date': 'Wed Jul 15 2020 19:49:30.569 GMt-0300 (Hora padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 87
  self.ambiente = Cena(self.matrizMapaFase[0][0])
  ^
IndentationError: unexpected indent
'''},
{'date': 'Wed Jul 15 2020 19:49:32.284 GMt-0300 (Hora padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 87
  self.ambiente = Cena(self.matrizMapaFase[0][0])
  ^
IndentationError: unexpected indent
'''},
{'date': 'Wed Jul 15 2020 20:25:36.789 GMt-0300 (Hora padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 80
  if se cont > 3:
         ^
SyntaxError: invalid syntax
'''},
{'date': 'Thu Jul 16 2020 09:41:07.496 GMt-0300 (Hora padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 53
  self.boneco = Elemento(self.BONECO, x=self.xx, y=self.yy = 40, cena=self.ambiente)
                                                                                    ^
SyntaxError: invalid syntax
'''},
{'date': 'Thu Jul 16 2020 09:56:46.844 GMt-0300 (Hora padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 177
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 96
    Eventos().vai()
  module <module> line 53
    self.boneco = Elemento(self.BONECO, x=self.x1, y=self.y1, cena=self.ambiente)
AttributeError: 'Eventos' object has no attribute 'y1'
'''},
{'date': 'Thu Jul 16 2020 09:59:41.827 GMt-0300 (Hora padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 50
  global matrizFase
  ^
IndentationError: expected an indented block
'''},
{'date': 'Thu Jul 16 2020 09:59:55.22 GMt-0300 (Hora padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 177
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 97
    Eventos().vai()
  module <module> line 51
    self.x1 = matrizFase[0][0]
NameError: name 'matrizFase' is not defined
'''},
{'date': 'Thu Jul 16 2020 09:59:58.629 GMt-0300 (Hora padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 177
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 97
    Eventos().vai()
  module <module> line 51
    self.x1 = matrizFase[0][0]
NameError: name 'matrizFase' is not defined
'''},
{'date': 'Thu Jul 16 2020 10:00:10.172 GMt-0300 (Hora padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 177
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 97
    Eventos().vai()
  module <module> line 51
    self.x1 = matrizFase[0][1]
NameError: name 'matrizFase' is not defined
'''},
{'date': 'Thu Jul 16 2020 10:00:14.165 GMt-0300 (Hora padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 177
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 97
    Eventos().vai()
  module <module> line 51
    self.x1 = matrizFase[0][1]
NameError: name 'matrizFase' is not defined
'''},
{'date': 'Thu Jul 16 2020 10:01:07.462 GMt-0300 (Hora padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 177
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 30
    matrizFase = [[link2,60,260],     #matrizFase = [[local_imagem_fase, x_inicial, y_inicial]...]
NameError: name 'link2' is not defined
'''},
{'date': 'Thu Jul 16 2020 10:11:53.628 GMt-0300 (Hora padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 177
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 30
    matrizFase = [[link2,60,260],     #matrizFase = [[local_imagem_fase, x_inicial, y_inicial]...]
NameError: name 'link2' is not defined
'''},
{'date': 'Thu Jul 16 2020 11:50:21.126 GMt-0300 (Hora padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 87
  if cont = 2: #Regulador do contador. Precisa alterar a programação para voltar a fase em um portal de retorno
           ^
SyntaxError: invalid syntax
'''},
{'date': 'Thu Jul 16 2020 12:39:32.800 GMt-0300 (Hora padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 77
  self.ambiente = Cena(self."https://i.imgur.com/5Qno2fs.png")#(self.matrizFase[0][0]) #lê a cena que está descrita na primeira coluna da matriz
                             ^
SyntaxError: invalid syntax
'''},
{'date': 'Thu Jul 16 2020 12:40:06.17 GMt-0300 (Hora padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 77
  self.ambiente = Cena(self."https://i.imgur.com/5Qno2fs.png")#(self.matrizFase[0][0]) #lê a cena que está descrita na primeira coluna da matriz
                             ^
SyntaxError: invalid syntax
'''},